WordPress/build.gradle

```diff
@@ -46,15 +46,18 @@ android {
         vectorDrawables.useSupportLibrary = true
 
         buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "true"
+        buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "true"
     }
 
     productFlavors {
         vanilla { // used for release and beta
             buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "false"
+            buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "false"
         }
 
         zalpha { // alpha version - enable experimental features
             applicationId "org.wordpress.android"
+            buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "false"
```

Comment:
Makes sense but this is a key feature, and if it breaks during development it will be a PITA for other devs (they can edit and disable the feature flag), that's why I think a feature branch is more adapted for this: if it breaks only you will be affected.